projects/wvr-elements/src/lib/wvr-nav-list/wvr-nav-li/wvr-nav-li.component.ts

```diff
@@ -1,19 +1,24 @@
-import { Component, Input } from '@angular/core';
-import { wvrThemeHostBindings, wvrThemeInputs } from '../../shared/wvr-styles';
-
+import { Component, HostBinding, Input } from '@angular/core';
+/**
+ * The WvrNavLi Component contains the href property of the list.
+ */
 @Component({
   selector: 'wvr-nav-li-element',
   templateUrl: './wvr-nav-li.component.html',
-  styleUrls: ['./wvr-nav-li.component.scss'],
-  inputs: [
-    ...wvrThemeInputs
-  ],
-  host: {
-    ...wvrThemeHostBindings
-  }
+  styleUrls: ['./wvr-nav-li.component.scss']
 })
 export class WvrNavLiComponent {
 
+  /** This property contains the href value of the list element. */
```

Comment:
A resolvable URI to to which this li will link, when provided. If no link is provided, the list element will not display as a link.

Reply:
Applied correction